docs/readthedocs/source/doc/Application/powered-by.md

```diff
@@ -82,5 +87,7 @@ Learning on Apache Spark and Analytics Zoo](https://www.dellemc.com/resources/en
 <br>• [Technical Verification of SGX and BigDL Based Privacy Computing for Multi Source Financial Big Data](https://www.intel.cn/content/www/cn/zh/now/data-centric/sgx-bigdl-financial-big-data.html) (in Chinese)
 * __World Bank__
 <br>• [Using Crowdsourced Images to Create Image Recognition Models with Analytics Zoo using BigDL](https://databricks.com/session/using-crowdsourced-images-to-create-image-recognition-models-with-bigdl)
-* __Yunda__
+*  __Yahoo!__
```

Comment:
Yahaoo Japan

Reply:
Confirmed by the customer, name has been corrected to Yahoo! JAPAN